addons/udes_stock/wizard/mark_todo_stock_move.py

```diff
@@ -0,0 +1,70 @@
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+
+
+class MarkTodoStockMove(models.TransientModel):
+    _name = "mark.todo.stock.move"
+    _description = "Update stock move in draft to Todo status"
```

Comment:
Suggestion: Linked to the above, maybe something like `Mark Transfers as Todo Wizard`? Again worth getting another opinion. :)